src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -282,6 +316,36 @@ export function setupTerminalMenus(): void {
 					},
 					group: ContextMenuGroup.Config
 				}
+			},
+			{
+				id: MenuId.TerminalEditorInstanceContext,
+				item: {
+					command: {
+						id: TerminalCommandId.RevealInFinder,
+						title: localize('workbench.action.terminal.revealInFinder', "Reveal Current Working Directory in Finder"),
+					},
+					group: ContextMenuGroup.Navigation,
+				}
+			},
+			{
+				id: MenuId.TerminalEditorInstanceContext,
+				item: {
+					command: {
+						id: TerminalCommandId.RevealInExplorer,
+						title: localize('workbench.action.terminal.revealInExplorer', "Reveal Current Working Directory in Explorer"),
```

Comment:
We want this to say "Explorer View" like tabs:

<img width="368" alt="Screenshot 2025-01-08 at 8 45 14 am" src="https://github.com/user-attachments/assets/b5549c61-c840-4efe-9b58-4c74aedb2304" />